### PR TITLE
update PKCS115_Cipher.decrypt annotation

### PR DESCRIPTION
**Pull request by arnimarj**
There are a couple of issues with the type annotation for `PKCS115_Cipher.decrypt` addressed in this PR:

1. the `sentinal` object may be returned as-is
2. `expected_pt_len` has a default value

### CODE REVIEW

File: lib/Crypto/Cipher/PKCS1_v1_5.pyi [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
     def __init__(self,
                  key: RsaKey,
                  randfunc: Callable[[int], bytes]) -> None: ...
     def can_encrypt(self) -> bool: ...
-    def can_decrypt(self) -> bool: ...
     def encrypt(self, message: Buffer) -> bytes: ...
     def decrypt(self, ciphertext: Buffer,
                 sentinel: Any,
-                expected_pt_len: Optional[int]) -> bytes: ...
+                expected_pt_len: Optional[int] = ...) -> Union[bytes, T]: ...
```
Comment (Varbin) [inline lib/Crypto/Cipher/PKCS1_v1_5.pyi:17]: Shouldn't line 16 be `sentinel: T`?

File: lib/Crypto/Cipher/PKCS1_v1_5.pyi [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 Buffer = Union[bytes, bytearray, memoryview]
+T = TypeVar('T')
+
 
 class PKCS115_Cipher:
     def __init__(self,
                  key: RsaKey,
                  randfunc: Callable[[int], bytes]) -> None: ...
     def can_encrypt(self) -> bool: ...
-    def can_decrypt(self) -> bool: ...
```
Comment (Varbin) [inline lib/Crypto/Cipher/PKCS1_v1_5.pyi:12]: What happened to `can_decrypt`?

**Review by Varbin — CHANGES_REQUESTED** [review]

File: lib/Crypto/Cipher/PKCS1_v1_5.pyi (hunk 2 above)
Comment (arnimarj) [reply lib/Crypto/Cipher/PKCS1_v1_5.pyi:12]: Sorry... a botched merge from my fork

File: lib/Crypto/Cipher/PKCS1_v1_5.pyi (hunk 1 above)
Comment (arnimarj) [reply lib/Crypto/Cipher/PKCS1_v1_5.pyi:18]: Correct. Fixed.

**Comment by Legrandin** [comment]
Merged, thanks!

**Pull request closed without merging**